src/main/java/frc/robot/subsystems/CoralSubsystem.java

```diff
@@ -0,0 +1,103 @@
+package frc.robot.subsystems;
+
+import bearlib.motor.deserializer.MotorParser;
+import com.revrobotics.spark.SparkBase;
+import edu.wpi.first.wpilibj.DigitalInput;
+import edu.wpi.first.wpilibj.Filesystem;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.Commands;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import java.io.File;
+import java.io.IOException;
+
+public class CoralSubsystem extends SubsystemBase {
+  private MotorParser intakeParser;
+  private SparkBase intake;
+  private DigitalInput intakeSensor = new DigitalInput(0);
+
+  /**
+   * Constructs a CoralSubsystem. Coral Subsystem fully develops the low level architecture for the
+   * Coral Intake. Coral Subsystem also provides functionality for full high level control over the
+   * Intake
+   */
+  public CoralSubsystem() {
+    configureMotors();
+  }
+
+  /**
+   * pulls the Json motor files and passes them into the motor wrapper/parser
+   *
+   * @throws RuntimeException if the motor configuration fails
+   */
+  private void configureMotors() {
+    File directory = new File(Filesystem.getDeployDirectory(), "Elevator");
+    try {
+      intakeParser = new MotorParser(directory).withMotor("intakeMotor.json");
+      this.intake = this.intakeParser.configure().getSpark();
+    } catch (IOException e) {
+      throw new RuntimeException("Failed to configure motors: " + e.getCause().toString());
```

Comment:
We should pass this to RuntimeException's second parameter, ``cause``

```suggestion
    } catch (IOException exception) {
      throw new RuntimeException("Failed to configure coral motor!", exception);
```